granule_ingester/granule_ingester/pipeline/Pipeline.py

```diff
@@ -48,36 +53,56 @@
 
 
 def _init_worker(processor_list, dataset, data_store_factory, metadata_store_factory, shared_memory):
-    global _worker_data_store
-    global _worker_metadata_store
     global _worker_processor_list
     global _worker_dataset
     global _shared_memory
 
     # _worker_data_store and _worker_metadata_store open multiple TCP sockets from each worker process;
     # however, these sockets will be automatically closed by the OS once the worker processes die so no need to worry.
-    _worker_data_store = data_store_factory()
-    _worker_metadata_store = metadata_store_factory()
     _worker_processor_list = processor_list
     _worker_dataset = dataset
     _shared_memory = shared_memory
 
+    logger.debug("worker init")
 
 async def _process_tile_in_worker(serialized_input_tile: str):
     try:
+        logger.info('Starting tile creation subprocess')
         logger.debug(f'serialized_input_tile: {serialized_input_tile}')
         input_tile = nexusproto.NexusTile.FromString(serialized_input_tile)
-        logger.debug(f'_recurse params: _worker_processor_list = {_worker_processor_list}, _worker_dataset = {_worker_dataset}, input_tile = {input_tile}')
-        processed_tile = _recurse(_worker_processor_list, _worker_dataset, input_tile)
+        # logger.debug(f'_recurse params: _worker_processor_list = {_worker_processor_list}, _worker_dataset = {_worker_dataset}, input_tile = {input_tile}')
```

Comment:
Can you remove this line? There's a similarly commented out line above we should also remove

Reply:
Sure.